db/migrate/20180714153306_remove_comments_without_user.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveCommentsWithoutUser < ActiveRecord::Migration[5.1]
+  def up
+    execute <<~SQL
+      DELETE
+      FROM comments
+      WHERE user_id NOT IN (SELECT id FROM users)
+    SQL
+  end
```

Comment:
Can't check it our locally myself right now, but I'm curious: what happens to migrations that do not have a `down` method?

Reply:
You can still roll them back - which changes the schema version as normal - but they just don't do anything 🤷‍♂️